services/service-core/src/graphql/users/test/users.service.spec.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Humanitech Supply Trail
+ *
+ * Copyright (c) Humanitech, Peter Rogov and Contributors
+ *
+ * Website: https://humanitech.net
+ * Repository: https://github.com/humanitech-net/supply-trail
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Test, TestingModule } from '@nestjs/testing';
+import { UsersService } from '../users.service';
+
+describe('UsersSerivce', () => {
+  let usersService: UsersService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [UsersService]
+    }).compile();
+    usersService = module.get<UsersService>(UsersService);
+  });
+
+  it('', () => {
```

Comment:
Test must have a descriptive name. Something like `it('must have userService defined')`